packages/nuxt/src/common/snippets.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+// todo: tests
+/** Returns an import snippet */
+export function buildSdkInitFileImportSnippet(filePath: string): string {
+  const pathToPosix = (originalPath: string): string => {
+    return originalPath.split(path.sep).join(path.posix.sep);
+  };
+
+  return `import "${pathToPosix(filePath)}";`;
+}
+
+// todo: tests
+/** Adds an import statement right after <script setup> */
+export function addImportStatement(filePath: string, importStatement: string): void {
```

Comment:
m: Shouldn't we try to put the import statement for Sentry as high as possible? This means we can have some scripts that occur before sentry gets initialized.

Reply:
Maybe my comment is misleading. Sentry is the first import after `<script setup>` in `nuxt-root.vue` (but maybe there is an even "higher" file). So this would look like this:

```html
<template>
 <!-- Suspense etc. ->
 </ template>

<script setup>
import "very/long/path/nuxt/sentry.client.config.ts";
// other imports
```